<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CandleStick_Box_adv</class>
 <widget class="QWidget" name="CandleStick_Box_adv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CandleStick_Box</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QChartView" name="chartView" native="true"/>
        </item>
        <item>
         <widget class="QChartView" name="volumes_widget" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="lbl_volumes_grapher"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>zoom</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QDoubleSpinBox" name="dsb_zoom"/>
             </item>
             <item>
              <widget class="QToolButton" name="btn_zoom_in">
               <property name="text">
                <string>-</string>
               </property>
               <property name="icon">
                <iconset resource="../../../../../icons/digits/digits.qrc">
                 <normaloff>:/plus_minus/minus.png</normaloff>:/plus_minus/minus.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_zoom_out">
               <property name="text">
                <string>+</string>
               </property>
               <property name="icon">
                <iconset resource="../../../../../icons/digits/digits.qrc">
                 <normaloff>:/plus_minus/plus.png</normaloff>:/plus_minus/plus.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_zoom_reset">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="cb_rubberband"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>move</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSpinBox" name="sb_pos"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QToolButton" name="btn_move_down">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../../../../../lib/pictures/arrow.qrc">
                 <normaloff>:/arrow/arrow-down.png</normaloff>:/arrow/arrow-down.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_move_up">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../../../../../lib/pictures/arrow.qrc">
                 <normaloff>:/arrow/arrow-up.png</normaloff>:/arrow/arrow-up.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_move_left">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../../../../../lib/pictures/arrow.qrc">
                 <normaloff>:/arrow/arrow-left.png</normaloff>:/arrow/arrow-left.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_move_right">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../../../../../lib/pictures/arrow.qrc">
                 <normaloff>:/arrow/arrow-right.png</normaloff>:/arrow/arrow-right.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btn_move_reset">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>themes</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QToolButton" name="btn_theme_light">
             <property name="text">
              <string>L</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_theme_dark">
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_test">
          <property name="text">
           <string>test</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>38</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_clear_data">
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="btn_show_frame">
         <property name="text">
          <string>...</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dsb_zoom</tabstop>
  <tabstop>btn_zoom_in</tabstop>
  <tabstop>btn_zoom_out</tabstop>
  <tabstop>btn_zoom_reset</tabstop>
  <tabstop>cb_rubberband</tabstop>
  <tabstop>sb_pos</tabstop>
  <tabstop>btn_move_down</tabstop>
  <tabstop>btn_move_up</tabstop>
  <tabstop>btn_move_left</tabstop>
  <tabstop>btn_move_right</tabstop>
  <tabstop>btn_move_reset</tabstop>
  <tabstop>btn_theme_light</tabstop>
  <tabstop>btn_theme_dark</tabstop>
  <tabstop>btn_test</tabstop>
  <tabstop>btn_clear_data</tabstop>
  <tabstop>btn_show_frame</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../../lib/pictures/arrow.qrc"/>
  <include location="../../../../../icons/digits/digits.qrc"/>
 </resources>
 <connections/>
</ui>
